docker-compose.yml

```diff
@@ -1,5 +1,26 @@
 version: "3.9"
 services:
+  coturn:
+    environment:
+      REALM: ret
+      PSQL: host=db dbname=ret_dev user=postgres password=postgres options='-c search_path=coturn' connect_timeout=30
```

Comment:
Does this need access to Reticulum’s database? If not, it should get its own and probably its own host too since the version of Postgres Reticulum needs may differ from that of coturn.